download.sh

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 # This software may be used and distributed according to the terms of the GNU General Public License version 3.
 
-PRESIGNED_URL=""             # replace with presigned url from email
-MODEL_SIZE="7B,13B,30B,65B"  # edit this list with the model sizes you wish to download
-TARGET_FOLDER=""             # where all files should end up
+PRESIGNED_URL="${PRESIGNED_URL:-}"              # get it from email and hard code it here, pass as env PRESIGNED_URL or as parameter
+MODEL_SIZE="${MODEL_SIZE:-7B,13B,30B,65B}"      # edit this list with the model sizes you wish to download
+TARGET_FOLDER="${TARGET_FOLDER:-model-weights}" # read an env with TARGET_FOLDER where all files should end up or hard code it here
 
 declare -A N_SHARD_DICT
```

Comment:
There have been changes to main, including how the models are declared and prompting the user for a URL.  If you merge in these updates, a possible solution might be to check to see if the promptable variables (PRESIGNED_URL, MODEL_SIZE) are emppty prior to prompting.  This way, if someone provides appropriate ENV vars, the script can be noninteractive.  If they do not provide ENV vars, the current behavior of prompting them will be preserved.